src/useReducedMotion/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+
+export const useReducedMotion = () => {
```

Comment:
<img width="350" alt="image" src="https://github.com/cargurus/transmish/assets/56884088/b316dd58-c985-4446-bd6c-b48e9c8a0262">

The piece of information above is so important, I think it's worth putting in JSDoc format on the hook. SSR is so new for lots of devs that it's worth doing a little handholding

```suggestion
/**
 * This will always return false when doing SSR
 * or something like that, use your judgment of how to word it
 */
export const useReducedMotion = () => {
```